sed: 1: "/^EXAMPLE/,/FILES/ { /E ...": extra characters at the end of p command
--
--- a/samples/extract-example.sh
+++ b/samples/extract-example.sh
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-cat $1 | sed -ne '/^EXAMPLE/,/FILES/ { /EXAMPLE/,+2d ; /^FILES/d ;  /^--/d ; p}' > $2
+cat $1 | gsed -ne '/^EXAMPLE/,/FILES/ { /EXAMPLE/,+2d ; /^FILES/d ;  /^--/d ; p}' > $2
